### DISCUSSION TITLE
daily/weekly build of typst for windows?

### DISCUSSION THREAD

**Discussion by retsyo** [typst/typst#1170] [Q&A]
I am using Msys2+Mingw64 on windows 10 64bits. I failed to install/build required version of rust in it, as a result I can't compile typst by myself. So is there and daily, weekly, or bi-weekly build of typst for windows?

thanks

**Comment by PgBiel** [comment]
Typst is automatically built for multiple platforms (including windows) on every release, check it out: https://github.com/typst/typst/releases/

**Comment by ghost** [comment]
I also support this because of long compile times. Though, Rust and Typst can be best installed using the following on Windows 10 x64:
1. Download and run `https://static.rust-lang.org/rustup/dist/x86_64-pc-windows-msvc/rustup-init.exe`
2. Then `git clone https://github.com/typst/typst.git`
3. `cd typst`
4. `cargo build --release`
5. Wait 2 to 5 minutes
6. The binary is now in `typst\target\release\typst.exe`

Maybe it is possible to include the compiled outputs from the CI directly as artifacts.

**Reply by retsyo** [reply]
> Though, Rust and Typst can be best installed using the following on Windows 10 x64:
> 1. Download and run `https://static.rust-lang.org/rustup/dist/x86_64-pc-windows-msvc/rustup-init.exe`

the problem is, I am using MSYS2+MingW64 on windows 10 64bits. There is no more space for yet another Visual Stduio Code :(

**Reply by ghost** [reply]
This one can maybe help `https://static.rust-lang.org/dist/rust-1.69.0-x86_64-pc-windows-gnu.msi` from [here](https://forge.rust-lang.org/infra/other-installation-methods.html#standalone-installers)
